src/utils/calendar.ts

```diff
@@ -0,0 +1,34 @@
+import { PersonalizedVenueEvent } from "types/venues";
+import { VenueEvent } from "types/venues";
+import { eventEndTime, eventStartTime } from "utils/event";
+import { WithVenueId } from "utils/id";
+import { getFullVenueInsideUrl } from "utils/url";
+
+import ical from "ical-generator";
+
+export const createCalendar = (
+  allEvents: PersonalizedVenueEvent[] | WithVenueId<VenueEvent>[],
+  calendarname: string
+) => {
+  const cal = ical({ name: calendarname });
+
+  if (!cal) return;
+
+  allEvents.map((event: PersonalizedVenueEvent | WithVenueId<VenueEvent>) =>
+    cal.createEvent({
+      id: event.venueId,
+      start: eventStartTime(event),
+      end: eventEndTime(event),
+      description: event.host,
```

Comment:
Might be able to put `event.host` in `organizer`?

- https://sebbo2002.github.io/ical-generator/develop/reference/classes/icalevent.html#organizer

---

You can find the `VenueEvent` type definition in `src/types/venues.ts`

It has a `description` field, so that would probably be good to include here.

- https://sebbo2002.github.io/ical-generator/develop/reference/classes/icalevent.html#description

---

```suggestion
      organizer: event.host,
      description: event.description,
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1423/commits/ec68e834fab42e9549a285d7a454e4c29e1f24e4